### PR TITLE
Added csscomb and added some helpers from nikta/blank

### PR DESCRIPTION
**Pull request by aduarteraposo**
Added csscomb as configurable extra and added some helpers from nikita/blank.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -502,6 +507,17 @@ var NikitaGenerator = yeoman.generators.Base.extend({
 			{
 				delete packageJsonData['devDependencies']['grunt-sass'];
 			}
+
+			// CSS-Comb
+			if (this.config.get('features').indexOf('csscomb') !== -1)
```
Comment (DracoBlue) [inline app/index.js]: Duplicate line

File: app/index.js [hunk 2]
```diff
@@ -396,6 +400,7 @@ var NikitaGenerator = yeoman.generators.Base.extend({
 			// Standard Files & Folders
 			this.template('.gitignore.ejs', '.gitignore');
 			this.template('.sass-lint.yml.ejs', '.sass-lint.yml');
+			this.template('.csscomb.json', '.csscomb.json');
```
Comment (DracoBlue) [inline app/index.js]: csscomb line is already in IF, so this should be removed

File: app/templates/Gruntfile.js.ejs [hunk 3]
```diff
@@ -134,7 +134,20 @@ module.exports = function(grunt) {
...
+
+		// Configuration for csscomb
+		csscomb: {
+			options: {
+				config: '.csscomb.json'
+			},
+			dev: {
+				expand: true,
+				cwd: '<%%= paths.src %>/sass/',
+				src: ['**/_*.scss', '!mixins/*.scss', '!variables/*.scss'],
```
Comment (DracoBlue) [inline app/templates/Gruntfile.js.ejs]: Please *.scss and not _*.scss

File: app/templates/README.md.ejs [hunk 4]
```diff
@@ -90,7 +90,8 @@ You don't like to stare permanently on your console? So wouldn’t it be great i
 <% if (config.get('staticPageGenerator').indexOf('assemble') != -1) { %>
 - [assemble](https://github.com/assemble/assemble)<% } %>
 - [autoprefixer-core](https://github.com/postcss/autoprefixer-core)<% if (config.get('features').indexOf('bower') != -1) { %>
-- [bower](https://github.com/bower/bower)<% } %>
+- [bower](https://github.com/bower/bower)<% } %><% if (config.get('features').indexOf('csscomb') != -1) { %>
+- [csscomb](https://github.com/csscomb/csscomb.js)<% } %>
```
Comment (DracoBlue) [inline app/templates/README.md.ejs]: Please feature flag for csscomb here

File: app/templates/source/sass/mixins/_helpers.scss.ejs [hunk 5]
```diff
@@ -0,0 +1,35 @@
+//
+// layout helpers
```
Comment (DracoBlue) [inline app/templates/source/sass/mixins/_helpers.scss.ejs]: file should be part of nikita css

File: app/templates/source/sass/mixins/_respond-to-optional.scss.ejs [hunk 6]
```diff
@@ -0,0 +1,38 @@
+//
+// breakpoint mixin with optional fallback
+// --------------------
+//
+//
+
```
Comment (DracoBlue) [inline app/templates/source/sass/mixins/_respond-to-optional.scss.ejs]: Please replace the original version. No need for 2 versions with same name

**Comment by axten** [comment]
we should not go with csscomb. The tool is no longer being developed.
@aduarteraposo please open a PR on nikita/css for adding the mixins.

**Pull request closed without merging**